cmd/watch.go

```diff
@@ -28,9 +28,9 @@ var watchCmd = &cobra.Command{
 }
 
 func init() {
-	SkuidCmd.AddCommand(watchCmd)
 	flags.AddFlags(watchCmd, flags.NLXLoginFlags...)
 	flags.AddFlags(watchCmd, flags.Directory)
+	AppCmd = append(AppCmd, retrieveCmd)
```

Comment:
@gzapatas Shouldn't this be appending the watchCmd?

Reply:
Nice catch